selfservice/strategy/password/validator.go

```diff
@@ -172,7 +166,10 @@ func (s *DefaultPasswordValidator) Validate(identifier, password string) error {
 	s.RUnlock()
 
 	if !ok {
-		if err := s.fetch(hpw); err != nil {
+		if shouldIgnore, err := s.fetch(hpw); err != nil {
+			if shouldIgnore {
+				return nil
+			}
```

Comment:
And then you do this:

```suggestion
		if err := s.fetch(hpw); errors.Is(err, ErrNetworkFailure) {
			return nil
		} else if err != nil {
			return err
		}
```